plugins/kubernetes-backend/src/service/KubernetesFetcher.ts

```diff
@@ -249,7 +248,7 @@ export class KubernetesClientBasedFetcher implements KubernetesFetcher {
     return (
       authProvider === 'serviceAccount' &&
       !clusterDetails.authMetadata.serviceAccountToken &&
-      fs.pathExistsSync(Config.SERVICEACCOUNT_CA_PATH)
+      fs.pathExistsSync('/var/run/secrets/kubernetes.io/serviceaccount/ca.crt')
```

Comment:
is there any reason why this is not taken from `@kubernetes/client-node` anymore?

Reply:
I see! Thank you for pointing it out. Since we are here I think this line should be changed to:

```suggestion
      await fs.pathExists('/var/run/secrets/kubernetes.io/serviceaccount/ca.crt')
```

so that we don't block the event loop. But the suggestion isn't related to your PR, so up to you if you want to address now

Reply:
Included in [8d6eb38](https://github.com/backstage/backstage/pull/23320/commits/8d6eb389819cfc6706f239f96e9d59aec1fbc7ba)